flink-runtime/src/main/java/org/apache/flink/runtime/metrics/ReporterSetup.java

```diff
@@ -178,36 +164,96 @@ private static ReporterSetup createReporterSetup(String reporterName, MetricConf
 				metricReporterOptional.ifPresent(reporter -> {
 					MetricConfig metricConfig = new MetricConfig();
 					reporterConfig.addAllToProperties(metricConfig);
-
-					reporterArguments.add(createReporterSetup(reporterName, metricConfig, reporter));
+					reporterSetups.add(createReporterSetup(reporterName, metricConfig, reporter));
 				});
-			}
-			catch (Throwable t) {
+			} catch (Throwable t) {
```

Comment:
revert

Reply:
Is this an accepted style in Flink? I mostly see "classic" variant with `} catch ...` (including the same class in loadReporterFactories() method.

Reply:
This code was touched anyhow, because of refactoring, so I think it should be OK to do such things, unless you have a strong opinion about this.

Reply:
our code style unfortunately does not cover the placement of such braces; hence we reject any changes such as this to existing code.

Reply:
@zentol Ok, I find it a bit strange, but I am not setting the rules here.

Reply:
Done.